Ginger/GingerCoreNET/WorkSpaceLib/WorkSpace.cs

```diff
@@ -374,6 +379,7 @@ public bool OpenSolution(string solutionFolder)
                     {
                         //Reporter.ToUser(eUserMsgKey.BeginWithNoSelectSolution);
                         Reporter.ToLog(eLogLevel.WARN, "Loading Solution- Error: Solution File Not Found");
+                        Console.WriteLine("Solution File not exist");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4707297)